bundles/org.openhab.binding.webexteams/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="webexteams"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Account Thing Type -->
+	<thing-type id="account">
+		<label>WebexTeams Account</label>
+		<description>WebexTeams account used to send messages.</description>
+
+		<channels>
+			<channel id="botname" typeId="botname"/>
+			<channel id="status" typeId="status"/>
+			<channel id="lastactivity" typeId="lastactivity"/>
+		</channels>
+
+		<properties>
+			<property name="personType">Unknown type</property>
+			<property name="name">Unknown name</property>
+		</properties>
+
+		<config-description>
+			<parameter name="token" type="text" required="false">
+				<context>password</context>
+				<label>Token</label>
+				<description>Authentication token (Bot)</description>
+			</parameter>
+			<parameter name="clientId" type="text" required="false">
+				<label>Client Id</label>
+				<description>OAuth client Id (Integration)</description>
+			</parameter>
+			<parameter name="clientSecret" type="text" required="false">
+				<context>password</context>
+				<label>Client Secret</label>
+				<description>OAuth client secret (Integration)</description>
+			</parameter>
+			<parameter name="authCode" type="text" required="false">
+				<label>Auth Code (Integration)</label>
+				<description>Get the auth code from https://developer.webex.com/my-apps
+					by navigating to OAuth Authorization URL</description>
+			</parameter>
+			<parameter name="refreshToken" type="text" required="false">
+				<label>RefreshToken</label>
+				<context>password</context>
+				<advanced>true</advanced>
+				<description>Used internally. No need to manually set.</description>
+			</parameter>
+			<parameter name="roomId" type="text" required="false">
+				<label>Default room Id</label>
+				<description>Id of the default room to send messages</description>
+			</parameter>
+			<!--> <parameter name="refreshInterval" type="integer" unit="s" min="1">
+				<label>Refresh Interval</label>
+				<description>Interval the device is polled in sec.</description>
+				<default>600</default>
+				<advanced>true</advanced>
+				</parameter> -->
+		</config-description>
+	</thing-type>
+
+	<!-- Botname Channel Type -->
+	<channel-type id="botname">
+		<item-type>String</item-type>
+		<label>Bot Name</label><!-- Use uppercase words, except prepositions. 2-3 words, max 25 chars -->
```

Comment:
Please remove the comment on each channel type.

Reply:
Done.